### ISSUE TITLE
Keyboard events are always consumed when the screen is in full screen

### ISSUE THREAD

**Issue by yurivict** [pikvm/pikvm#1206]
**Is your feature request related to a problem? Please describe.**

Please allow the user to toggle the connection option 'Grab all keyboard events' when the screen is in a fullscreen mode.
This way the user would be able to "glance" into the connection with Alt-Tab and go away with another Alt-Tab, and not get stuck in that screen.

Remmina does this right: It allows the user to toggle the 'Grab all keyboard events' option with the button and the keyboard shortcut.

**Describe the solution you'd like**
See above.

**Describe alternatives you've considered**
n/a

**Additional context**
This feature is very useful when the user shares time between different systems and needs to be able to easily go back and forth.

**Comment by Kolyunya** [comment]
I'm using Chrome in the full-screen `application` mode on MacOS, and unfortunately, `Command+Tab` triggers window switch on the host machine instead of the remote one. Is there a way to cycle through windows of the remote machine using a standard keyboard shortcut?

**Comment by mdevaev** [comment]
> Is there a way to cycle through windows of the remote machine using a standard keyboard shortcut?

No. It's a big pain becuse some keys on macos cannot be intercepted in the browser at all.

**Comment by mdevaev** [comment]
@Kolyunya here is the workaround: https://docs.pikvm.org/shortcuts/

**Comment by mdevaev** [comment]
@yurivict I have released very large UI updates in full-screen mode (the panel is now available there) and wanted to make your request too, but found that it was impossible.

* [Keyboard Lock API on Blink-based browsers](https://developer.chrome.com/docs/capabilities/web-apis/keyboard-lock?) doesn't allow you to filter hotkeys, only single buttons, so it's generally useless for your feature-request.
* [Mozilla's proposal](https://github.com/mozilla/standards-positions/issues/196) allows you to limit hotkeys but only with some pre-defined subset (but they even still not implemented this).
* [Same for WebKit/Safari](https://github.com/WebKit/standards-positions/issues/182).
* There is now way to re-raise/emit key events to OS from the browser.

Sorry :(
